app/services/gobierto_people/agendas_integration.rb

```diff
@@ -0,0 +1,13 @@
+module GobiertoPeople
+  class AgendasIntegration
+
+    def self.sync_all_agendas
```

Comment:
What about renaming this method to `sync`?

- we don't need agendas/calendars because it's implicit in the class name
- we don't need the all because is a method class and is implicit too